inc/modules/export/epub/class-epub.php

```diff
@@ -844,30 +856,32 @@ protected function fetchAndSaveUniqueMedia( $url, $fullpath ) {
 		fclose( $GLOBALS[ $resource_key ] ); // @codingStandardsIgnoreLine
 
 		$this->fetchedMediaCache[ $url ] = $filename;
+
 		return $filename;
 	}
 
 	/**
 	 * Clean up content processed by HTML5 Parser, change it back into XHTML
 	 *
-	 * @param $html
+	 * @param string $html
 	 *
 	 * @return string
 	 */
-	protected function html5ToXhtml( $html ) {
+	protected function html5ToXhtml( string $html ): string {
```

Comment:
I think that this method could be removed since we want to have a valid HTML5. Might be worth to check this in the future